src/components/checkBox.tsx

```diff
@@ -30,11 +40,6 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
     alignItems: 'center',
     flex: 1,
-    marginBottom: 1.5,
```

Comment:
Removal of this margin isn't going to cause items butted up too close together?

Reply:
No there is plenty of padding on Checkbox.Item already no need for additional.